openquake/engine/engine.py

```diff
@@ -35,7 +35,7 @@
 TERMINATE = valid.boolean(
     config.get('distribution', 'terminate_workers_on_revoke') or 'false')
 
-USE_CELERY = config.get('distribution', 'oq_distribute') == 'celery'
+USE_CELERY = os.environ.get('OQ_DISTRIBUTE') == 'celery'
```

Comment:
Shouldn't be both? From env and from config?

Reply:
No, look inside `openquake.commands.__main__`: the environment variable is set from the config file if the user does not specify it. It is perverse, but I could not find another way to transmit the information to baselib .parallel which has no access to the configuration file, which is in commonlib.